client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -1,81 +1,336 @@
 /**
```

Comment:
@mtias sure, `plan-setup` is pretty generic. would `plugins-setup` still be too generic? `plugins-jetpack-setup`? trying to follow the convention of the neighbor folders with the `plugins-` prefix.